docs/data/pages.ts

```diff
@@ -1,8 +1,15 @@
 const pages = [
   {
-    pathname: '/x/advanced-components',
-    title: 'Overview',
+    pathname: '/x/getting-started/overview',
+    scopePathnames: ['/x/getting-started/overview'],
+    title: 'Getting started',
     icon: 'DescriptionIcon',
+    children: [
+      { pathname: `/x/getting-started/overview` },
+      { pathname: `/x/getting-started/licensing` },
+      { pathname: `/x/getting-started/support` },
+      { pathname: `/x/getting-started/roadmap` },
```

Comment:
Shouldn't they be one level up? I'm asking because both data grid and date pickers also have a "getting started" page, but in reality the user can only get started with the components itself, not with MUI X.

Reply:
That's an interesting question. It makes sense to me to group these pages together under a single header. Maybe we should call it something like "Introduction" rather than "Getting started"?

Reply:
+1 for "Introduction".

Reply:
I like Introduction, but I have to ask—will changing it cause a bunch of other things to break again? 😅

Reply:
Unfortunately yes. 😁 If you replace everything containing "getting-started" you may have less work, but inevitably something will break .

Reply:
Well in that case, let's make sure we don't need to change it again anytime soon. If anyone objects to "Introduction" then now is the time to speak up!

Reply:
What about calling the section header `Overview` and its first (index) page `Getting started`?

I'm suggesting that because I think we should also have on this index page a path to the "getting started" of each of the components.